app/javascript/tests/subject/monitoring_actions/MonitoringStatus.test.js

```diff
@@ -6,38 +6,17 @@ import MonitoringStatus from '../../../components/subject/monitoring_actions/Mon
 import InfoTooltip from '../../../components/util/InfoTooltip';
 import DateInput from '../../../components/util/DateInput';
 import { mockPatient1, mockPatient3 } from '../../mocks/mockPatients';
+import mockMonitoringReasons from '../../mocks/mockMonitoringReasons'
 
 const currentDate = moment(new Date()).format('YYYY-MM-DD');
 const newDate = moment(new Date('9-9-2020')).format('YYYY-MM-DD');
 const authyToken = 'Q1z4yZXLdN+tZod6dBSIlMbZ3yWAUFdY44U06QWffEP76nx1WGMHIz8rYxEUZsl9sspS3ePF2ZNmSue8wFpJGg==';
 const monitoringStatusValues = [ 'Actively Monitoring', 'Not Monitoring' ];
-const monitoringReasons = [
-  '--',
-  'Not Monitoring',
-  'Completed Monitoring',
-  'Meets criteria to shorten quarantine',
-  'Does not meet criteria for monitoring',
-  'Enrolled more than 14 days after last date of exposure (system)',
-  'Enrolled more than 10 days after last date of exposure (system)',
-  'Enrolled on last day of monitoring period (system)',
-  'Completed Monitoring (system)',
-  'Meets Case Definition',
-  'Lost to follow-up during monitoring period',
-  'Lost to follow-up (contact never established)',
-  'Transferred to another jurisdiction',
-  'Person Under Investigation (PUI)',
-  'Case confirmed',
-  'Past monitoring period',
-  'Meets criteria to discontinue isolation',
-  'Deceased',
-  'Duplicate',
-  'Other',
-  ' ',
-];
+const monitoringReasons = mockMonitoringReasons
 
 function getWrapper(patient, hasDependents, inHouseholdWithCeInExposure) {
   return shallow(<MonitoringStatus patient={patient} has_dependents={hasDependents} authenticity_token={authyToken}
-    in_household_with_member_with_ce_in_exposure={inHouseholdWithCeInExposure} monitoring_reasons={monitoringReasons.slice(1,monitoringReasons.length)}/>);
+    in_household_with_member_with_ce_in_exposure={inHouseholdWithCeInExposure} monitoring_reasons={monitoringReasons}/>);
```

Comment:
you can just pass in mockMonitoring reasons here, no need for the line a few above

Reply:
can you also add this in `MonitoringActions.test.js`?

Reply:
I did notice that, but intentionally made the decision to do it this way incase we ever wanted to modify what was in the mock compared to what we passed in. But I guess that'll probably never be necessary. Okay i'll change it.

Reply:
worst case if it needs to be modified someone can add that, nbd